node/chain_spec/src/lib.rs

```diff
@@ -1811,6 +1824,17 @@ fn mainnet_genesis(
         technical::Pallet::<Runtime>::tech_account_id_to_account_id(&dex_root_tech_account_id)
             .unwrap();
 
+    #[cfg(feature = "wip")]
```

Comment:
I'm not sure if it should be done this way. Should I remove the cfg at all?? Or we need to exclude WIP stuff from main chainspec

Reply:
I guess we should to hide the code above under the `wip` feature flag as well.